packages/react/src/react/components/compound/file-upload-field.tsx

```diff
@@ -172,13 +175,14 @@ export const FileUploadField = (props: FileUploadFieldProps) => {
     // TODO: Allow for multiples file, currently one at a time
     const targetFile = files[0]
 
-    const key = `${crypto.randomUUID()}-${targetFile.name}`
+    // I created a folder called prod-uploader to store the files. Do we need to change this to .env for custom folder?
+    const key = `prod-uploader/${crypto.randomUUID()}-${targetFile.name}`
+
+    const signedUrlPath =
+      storageAdapter?.grabPutObjectSignedUrlApiRoute?.path ?? '/api/storage/put-obj-signed-url'
```

Comment:
Check if storageAdapter is not defined, throw error. To guarantee that  grabPutObjectSignedUrlApiRoute should always provided